compression-service/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM maven:latest
+COPY . .
+
+EXPOSE 8080
+WORKDIR /EXML-converter-service
+#RUN ./buildJarEfficientXML.sh
+RUN ./buildJarEfficientXML.sh
+CMD ["java","-jar", "./target/EfficientXMLServer.jar" ]
```

Comment:
consider a [multistage build](https://docs.docker.com/develop/develop-images/multistage-build/) to only have the executable jar in the image